packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -93,7 +101,7 @@ export class Dropdown extends BaseComponent<IDropdownInternalProps, IDropdownSta
     return (
       <div ref='root'>
         { label && (
-          <label id={ id + '-label' } className='ms-Label' ref={ (dropdownLabel) => this._dropdownLabel = dropdownLabel } >{ label }</label>
+          <Label id={ id + '-label' } ref={ this._resolveRef('_dropdownLabel') } required={ required }>{ label }</Label>
```

Comment:
It is a div, but it's given the role of combobox, so as far as AX tools see it, it is an input control and not really a div. I wonder if something marked with a role of being an input control is labelable?